### PR TITLE
Feature/loop 18 notify friend

### PR DESCRIPTION
**Pull request by Lapeth**
A pane for inputting the friend to be notified.

### CODE REVIEW

File: template.php [hunk 1]
```diff
@@ -176,6 +176,11 @@ function loop_preprocess_panels_pane(&$variables) {
     $variables['theme_hook_suggestions'][] = 'panels_pane__editor';
   }
 
+  // Add template for "notify friend" box
```
Comment (rimi-itk) [inline template.php]: Please fix comment (cf. https://www.drupal.org/coding-standards/docs#inline)

Comment (Lapeth) [inline template.php]: What's wrong with it? It looks to me like the comment does follow the standard (as well as looking like the other inline comments in that function).

Comment (rimi-itk) [inline template.php]: All inline comments must end with a period.

Comment (Lapeth) [inline template.php]: Actually the doc doesn't state that. It states that comment blocks above functions must end in periods. But here you go:

**Pull request merged**